tests/diviner/test_diviner_model_export.py

```diff
@@ -304,13 +303,13 @@ def test_diviner_model_save_persists_specified_conda_env_in_mlflow_model_directo
     )
 
     pyfunc_conf = _get_flavor_configuration(model_path=model_path, flavor_name=pyfunc.FLAVOR_NAME)
-    saved_conda_env_path = os.path.join(model_path, pyfunc_conf[pyfunc.ENV])
-    assert os.path.exists(saved_conda_env_path)
-    assert saved_conda_env_path != diviner_custom_env
+    saved_conda_env_path = pathlib.Path(model_path).joinpath(pyfunc_conf[pyfunc.ENV])
+    assert saved_conda_env_path.exists()
+    assert str(saved_conda_env_path) != diviner_custom_env
 
     with open(diviner_custom_env, "r") as f:
         diviner_custom_env_parsed = yaml.safe_load(f)
-    with open(saved_conda_env_path, "r") as f:
+    with open(str(saved_conda_env_path), "r") as f:
```

Comment:
```suggestion
    with open(saved_conda_env_path, "r") as f:
```

`open` should work without `str`.